### PR TITLE
fix/severityFlag: Fixed --severity flag to take the user input value of severity

### PR DESCRIPTION
**Pull request by Ipuvi**
### What this does
This PR is linked to the issue raised [here](https://github.com/snyk-tech-services/jira-tickets-for-new-vulns/issues/115#issue-1250832749)

### Issue
- As per Readme, default value of severity was low while it was actually all severity i.e., severityThreshold
- If user passes a lower severity value, it was by default consider the value above it as well (for eg: medium -> {critical,high,medium} and high -> {critical,high}
- It should give user capability to use a particular severity as well

### Fix
As part of this PR, 
- I have fixed to get only user input value as severity along with severityThreshold
- To keep it consistent with existing behaviour kept the default values as all severity (critical,high,medium,low)
- If user wants multiple combination of severity, he/she can pass comma separated values to `--severity` optional flag

### Testing
As part of testing, I have tested by running all tests in my local and generating binary and executing same, also added and modified few test to verify the changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -131,7 +131,7 @@ jira:
     assigneeId: 123abc456def789
     assigneeName: AccountName
     priorityIsSeverity: true # <true|false>
-    label: label1 # <IssueLabel1>,<IssueLabel2>
```
Comment (akashvyom) [inline README.md:134]: @Ipuvi can we update the readme for the change in flags as well

File: vulns.go [hunk 2]
```diff
@@ -46,18 +47,43 @@ func getVulnsWithoutTicket(flags flags, projectID string, maturityFilter []strin
...
-		body.Filters.Severities = []string{"critical", "high"}
-	case "medium":
-		body.Filters.Severities = []string{"critical", "high", "medium"}
-	case "low":
-		body.Filters.Severities = []string{"critical", "high", "medium", "low"}
-	default:
-		log.Fatalln("Unexpected severity threshold")
-	}
+
+    if len(flags.optionalFlags.severities) > 0 && len(flags.optionalFlags.severityThreshold) == 0 {
```
Comment (akashvyom) [inline vulns.go]: @Ipuvi should we extract out this logic and call it getSeverties or something, this looks like being duplicated in both the methods

**Review by akashvyom — APPROVED** [review]
looks good

File: README.md (hunk 1 above)
Comment (Ipuvi) [reply README.md:134]: Done

File: vulns.go (hunk 2 above)
Comment (Ipuvi) [reply vulns.go]: Done

File: README.md (hunk 1 above)
Comment (Ipuvi) [reply README.md:134]: Done

**Pull request merged**